src/common/BackTrace.cc

```diff
@@ -51,23 +51,17 @@ std::string BackTrace::demangle(const char* name)
     int status;
     // only demangle a C++ mangled name
     if (mangled.compare(0, 2, "_Z") == 0) {
-      // just a guess, template names will go much wider
-      size_t len = 1024;
-      char* buf = (char*)malloc(len);
```

Comment:
Yeah, it's simpler and, likely, we don't need to care about optimizing this case at all. Simplicity won :+1: .